skill-tree/src/test/java/com/RDS/skilltree/integration/EndorsementsIntegrationTests.java

```diff
@@ -244,7 +244,7 @@ public void testAPIReturns200_OnEndorsementGivenId() {
                 .body("data.status", anyOf(equalTo("APPROVED"), equalTo("PENDING"), equalTo("REJECTED")))
                 .body("data.endorsementType", anyOf(equalTo("POSITIVE"), equalTo("NEGATIVE")))
                 .body("data.endorsersList", hasSize(1))
-                .body("data.endorsersList[0].endorserId", equalTo("user-2"))
+                .body("data.endorsersList[0].endorseeId", equalTo("user-2"))
```

Comment:
are we testing case where "data.endorsersList" can be null/empty

Reply:
Yes the lines just above this checks that as well